INSTALL.md

```diff
@@ -62,9 +62,29 @@ If desired, populate the configuration management tables by running the `cm_init
 
 ### Basic OS X PostgreSQL installation
 
-Follow the directions under `Interactive installer by EnterpriseDB` here: https://www.postgresql.org/download/macosx/
+Install the app found here: http://postgresapp.com/, and follow steps 1 and 2 on that page.
```

Comment:
I will leave it to the OSX users to decide if this app is the best option for getting things installed on OSX. But, we should be forward-looking with where the Web is going: the link here should be an `https://` URL, not unencrypted `http://`.